src/components/mock-lightspeed.ts

```diff
@@ -0,0 +1,26 @@
+import { Action, ExtensionHook } from '@openshift-console/dynamic-plugin-sdk';
```

Comment:
we can remove this as this will be implemented by other plugins

Reply:
Yes, this mock extension is for testing and is removed in the latest commit.